### PR TITLE
Db fix

### PR DESCRIPTION
**Pull request by joeray100**
- Merge branch 'main' of github.com:NoahZinter/find_my_city_be into main
- Merge branch 'models' into main
- Update schema

### CODE REVIEW

File: app/poros/city.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class CityDetail
+  attr_reader :categories,
+              :summary,
+              :total_score,
+              :city,
+              :state
+              # :picture - won't know till we get endpoint
+
+  def initialize(data)
+    @categories = data[:_embedded][:"city:search-results"][0][:_embedded][:"city:item"][:_embedded][:"city:urban_area"][:_embedded][:"ua:scores"][:categories]
```
Comment (NoahZinter) [inline app/poros/city.rb:10]: Wild api calls, great work Kon!

File: db/migrate/20210728233839_create_users.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
```
Comment (NoahZinter) [inline db/migrate/20210728233839_create_users.rb:1]: perfect

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+  create_table "favorites", force: :cascade do |t|
+    t.bigint "city_id"
+    t.bigint "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["city_id"], name: "index_favorites_on_city_id"
+    t.index ["user_id"], name: "index_favorites_on_user_id"
+  end
+
```
Comment (NoahZinter) [inline db/schema.rb:35]: looks great

**Review by NoahZinter — APPROVED** [review]
Thanks! Looks great

**Review by JoePeterson51 — APPROVED** [review]

**Pull request merged**